releases/2.4/release-notes.asciidoc

```diff
@@ -0,0 +1,116 @@
+---
+layout: release-notes
+title: Release Notes for Debezium 2.3
+debezium-version: "2.3"
+---
+:toc:
+:toc-placement: macro
+:toclevels: 1
+:sectanchors:
+:linkattrs:
+:icons: font
+
+All notable changes for Debezium releases are documented in this file.
+Release numbers follow http://semver.org[Semantic Versioning].
+
+toc::[]
+
+[[release-2.4.0-alpha1]]
+== *Release 2.4.0.Alpha1* _(July 14th 2023)_
+
+See the https://issues.redhat.com/secure/ReleaseNote.jspa?projectId=12317320&version=12409716[complete list of issues].
+
+=== Kafka compatibility
+
+This release has been built against Kafka Connect 3.5.0 and has been tested with version 3.5.0 of the Kafka brokers.
+See the https://kafka.apache.org/documentation/#upgrade[Kafka documentation] for compatibility with other versions of Kafka brokers.
+
+
+=== Upgrading
+
+Before upgrading any connector, be sure to check the backward-incompatible changes that have been made since the release you were using.
+
+When you decide to upgrade one of these connectors to 2.4.0.Alpha1 from any earlier versions,
+first check the migration notes for the version you're using.
+Gracefully stop the running connector, remove the old plugin files, install the 2.4.0.Alpha1 plugin files, and restart the connector using the same configuration.
+Upon restart, the 2.4.0.Alpha1 connectors will continue where the previous connector left off.
+As one might expect, all change events previously written to Kafka by the old connector will not be modified.
+
+If you are using our container images, then please do not forget to pull them fresh from https://hub.docker.com/u/debezium[DockerHub].
```

Comment:
Since 2.4 we should publish images only Quay.io. I guess we still publish them on Dockerhub, but we should probably advertise only quay.io images to slowly push users to use these images?

Reply:
@vjuranek :+1: